imap_processing/codice/tests/test_codice_l0.py

```diff
@@ -89,6 +103,25 @@ def test_housekeeping_data(
         # Compare raw values of other housekeeping data
         assert value.raw_value == validation_row[key]
 
+    # Determine the number of CCSDS header fields
+    num_ccsds_header_fields = 7
+
+    # Compare EU values of housekeeping data, skipping CCSDS header fields
+    for idx, field in enumerate(eu_hk_data):
+        # Skip the first num_ccsds_header_fields fields
+        if idx < num_ccsds_header_fields:
+            continue
+
+        # Extract the specific EU values for the field
+        eu_values = eu_hk_data[field].data
+
+        # Extract the corresponding validation data for the field
+        validation_values = validation_data[field].values
+
+        # Compare each individual element
+        for eu_val, validation_val in zip(eu_values, validation_values):
+            assert round(eu_val, 5) == round(validation_val, 5)
+
```

Comment:
Really nice work! 

You could move all above and this new changes into a new function because it's testing engineering unit of housekeeping. so you could call it like `test_eu_hk_data()` or something in that manner.